src/components/crownBoundaryEventDropdown.vue

```diff
@@ -53,10 +53,12 @@ export default {
     return {
       dropdownOpen: false,
       boundaryEventIcon,
+      noAvailableSpaceLabel: 'No available space',
```

Comment:
Could this be inlined instead?

Reply:
It made it trickier to read. But I can inline it.